CalibMuon/CSCCalibration/interface/CSCBadChambersConditions.h

```diff
@@ -25,7 +25,7 @@ class CSCBadChambersConditions: public edm::ESProducer, public edm::EventSetupRe
 
   inline static CSCBadChambers *  prefillBadChambers();
 
-  typedef const  CSCBadChambers * ReturnType;
+  typedef std::unique_ptr<CSCBadChambers> ReturnType;
```

Comment:
@gartung here and in other places you replace a pointer to a const object with a pointer to a non-const object.
Is this intended?

Reply:
In this particular case, it does not matter since the Framework takes ownership of the object and the framework does not allow the object to be changed.

Reply:
ok, i.e. it is intended in the sense that the new code still has the needed properties, but is more flexible as the user of the returned pointer can perform non-const operations. Is this correct?

Reply:
Correct. Just like when calling `edm::Event::put` in an `EDProducer` you pass a `std::unique_ptr<Thing>` instead of a `std::unique_ptr<const Thing>`.